### PR TITLE
Update anomaly_detection_unsupervised.ipynb

### PR DESCRIPTION
**Pull request by achoum**

### CODE REVIEW

File: docs/src/tutorials/anomaly_detection_unsupervised.ipynb [hunk 1]
```diff
@@ -157,180 +124,19 @@
    "source": [
     "## Load data\n",
     "\n",
-    "We use Pandas to load the data into a `DataFrame` and perform tabular manipulation before importing it into a Temporian as an `EventSet`.\n",
-    "\n",
-    "We index the data by the `machine` column in Temporian. This was, we can apply Temporian transformations on each machine individually as well as across all machines."
+    "We use Pandas to load the data into a `DataFrame`.\n",
+    "Often, data requires cleaning before it can be used, and Pandas si great in this task.\n",
```
Comment (rstz) [inline docs/src/tutorials/anomaly_detection_unsupervised.ipynb]: si great in this task. -> is great for this task

File: docs/src/tutorials/anomaly_detection_unsupervised.ipynb [hunk 2]
```diff
@@ -157,180 +124,19 @@
...
+    "However, in this dataset, the data is already cleansed, se we don't need to do any manipulation with Pandas."
```
Comment (rstz) [inline docs/src/tutorials/anomaly_detection_unsupervised.ipynb]: cleansed, se -> cleaned so

File: docs/src/tutorials/anomaly_detection_unsupervised.ipynb [hunk 3]
```diff
@@ -344,2008 +150,37 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b9f50a64-c27e-4cda-afd9-f5da6dcc0276",
+   "metadata": {},
+   "source": [
+    "Next, we convert the dataset into a Temporian `EventSet`. Temporian will help data visualization and feature computation.\n",
+    "\n",
+    "**Note:** Our anomaly detection model can operate directly on raw time series data, and that's how we'll begin. However, preprocessing and augmenting the temporal data (for example, calculating moving statistics or calendar events) before applying the anomaly detection model can significantly boost its performance. For example of feature engineering, check the [Feature engineering section](https://temporian.readthedocs.io/en/latest/tutorials/m5_competition/#feature-engineering) in the M5 Competition tutorial.\n"
```
Comment (rstz) [inline docs/src/tutorials/anomaly_detection_unsupervised.ipynb]: For example of feature engineering --> For an example of feature engineering

**Review by rstz — APPROVED** [review]
LGTM!

File: docs/src/tutorials/anomaly_detection_unsupervised.ipynb (hunk 1 above)
Comment (achoum) [reply docs/src/tutorials/anomaly_detection_unsupervised.ipynb]: *replaced

File: docs/src/tutorials/anomaly_detection_unsupervised.ipynb (hunk 2 above)
Comment (achoum) [reply docs/src/tutorials/anomaly_detection_unsupervised.ipynb]: *replaced

File: docs/src/tutorials/anomaly_detection_unsupervised.ipynb (hunk 3 above)
Comment (achoum) [reply docs/src/tutorials/anomaly_detection_unsupervised.ipynb]: *Replaced

**Comment by achoum** [comment]
Thanks!

**Pull request merged**